content/en/tracing/profiler/enabling/dotnet.md

```diff
@@ -41,6 +41,8 @@ Datadog .NET Profiler is currently in public beta. Datadog recommends evaluating
 
 Any language that targets the .NET runtime, such as C#, F#, and Visual Basic.
 
+Profiling is not supported on serverless platforms.
```

Comment:
NIT:

```suggestion
Profiling is not supported on serverless platforms such as AWS Lambda.
```

Reason: People might type something like "datadog profiling lambda" into google or Ctrl+F on the docs page. Not sure if Lambda is the main platform, but 3/3 Go customers who tried to use Go profiling on serverless where using lambda.

Reply:
One more nit- maybe s/Profiling/Continuous Profiler/. Kari, your call!

Reply:
Yep, I like those suggestions. I also nudged it around on a couple pages. Updated in a commit.